LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -59,6 +60,8 @@ public DeviceGetter(RegistryManager registryManager, ILoRaDeviceCacheStore cache
 
             try
             {
+                var devNonceBuffer = new byte[DevNonce.Size];
+                DevNonce? devNonce = Hexadecimal.TryParse(rawDevNonce, devNonceBuffer) ? DevNonce.Read(devNonceBuffer) : null;
```

Comment:
`Hexadecimal.TryParse` has an overload for `ulong`. Why not add one for `ushort` to make this easier and not need an array of two bytes for `devNonceBuffer`? Another option would be to use a local function to get around the limitation of not being able to use `Span<>` in `async` methods.